package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fontmin-webpack",
-  "version": "0.0.0",
```

Comment:
the version is managed by tags not the source, so no need to increment this it will happen automatically :)

Reply:
nice ! 👍

Reply:
can you please revert the change then :)

Reply:
Oups… 
Now it is reverted.